tests/integration/__init__.py

```diff
@@ -81,7 +81,7 @@
     # Avoid ${TMPDIR} and gettempdir() on MacOS as they yield a base path too long
     # for unix sockets: ``error: AF_UNIX path too long``
     # Gentoo Portage prefers ebuild tests are rooted in ${TMPDIR}
-    os.environ.get('TMPDIR', tempfile.gettempdir()) if salt.utils.is_darwin() else '/tmp'
+    os.environ.get('TMPDIR', tempfile.gettempdir()) if not salt.utils.is_darwin() else '/tmp'
```

Comment:
Ah, OK. If this is the intended behavior for both platforms then I'm good with it. Just wanted to make sure we weren't inadvertently modifying the logic. Thanks.